crates/biome_analyze/src/rule.rs

```diff
@@ -810,6 +810,22 @@ pub struct RuleAction<L: Language> {
     pub mutation: BatchMutation<L>,
 }
 
+impl<L: Language> RuleAction<L> {
+    pub fn new(
+        category: ActionCategory,
+        applicability: impl Into<Applicability>,
+        message: MarkupBuf,
```

Comment:
I think this can be changed into `message: Markup`, so we can move the `.to_owned()` inside this function, the caller won't need to do it anymore

Reply:
Based on [this SO answer](https://stackoverflow.com/a/72106272/3315164) I don't think `impl ToOwned` is the right way to go, at least for now. I think that would require significant changes to how `Markup` and `MarkupBuf` are implemented.

Reply:
Actually, we need `impl biome_console::fmt::Display`. 

Then inside the `new` function, we call `markup!().to_owned`